Makefile

```diff
@@ -0,0 +1,61 @@
+# Package related
+TOOLSDIR=$(CURDIR)/hack/tools/bin
+BUILDDIR=$(CURDIR)/build/_output
+
+
+# If gobin not set, create one on ./build and add to path.
+ifeq (,$(shell go env GOBIN))
+GOBIN=$(BUILD_DIR)/gobin
+else
+GOBIN=$(shell go env GOBIN)
+endif
+export PATH:=$(GOBIN):${PATH}
+
+$(TOOLSDIR):
+	@mkdir -p $@
+
+$(BUILDDIR): ; $(info Creating build directory...)
+	mkdir -p $@
+
+BRANCH ?= master
+# Paths to download the helm chart to.
+HELM_CHART_DEP_ROOT ?= $(BUILDDIR)/helmcharts
```

Comment:
just making sure:

if i run
`HELM_CHART_DEP_ROOT=/path/to/my/local/network-operator/deployment/network-operator make helm-docs`

it will work right ?

Reply:
yes, but there are two assumptions here:
- `HELM_CHART_DEP_ROOT` should contain your chart
- .helmdocsignore targets `$(BUILDDIR)/helmcharts`, so if you target another path, it won't be taken into account.

Reply:
can we add support for that ? does it make sense ?

what i was thinking:
1. dev works on some feature in network operator, eventually updates chart and submits PR
2. dev generates updated helm.rst from his local network operator branch and submits PR to network-operator-docs

Reply:
the content of the `.helmdocsignore` as to be relative it seems. I wanted to use `realpath` to make it works, but this isn't POSIX and have different behaviour between linux and mac... I don't have a solution so far for this.

Reply:
Maybe have another env PARAM like 'LOCAL_HELM_SOURCE_DIR`, and if it exists, copy content to build/_output/helmcharts/network-operator/charts ?

Reply:
let's take some time to think about this one. For now what I think is that a single repo would enable that easily :)

Reply:
let me followup with PR on generating from local sources. then each can pick an choose. thx for working on this one @souleb !